ipwb/replay.py

```diff
@@ -271,7 +271,8 @@ def showMemento(urir, datetime):
     else:
         resp = show_uri(uri, newDatetime)
 
-    resp.headers['Link'] = linkHeader
+    resp.headers.add('Link', linkHeader)
```

Comment:
This is inconsistent with how other instances of `resp.headers` are updated in the replay. Perhaps you should investigate why it works in other places without and explicit `.add()` method, but not here.

Reply:
I agree, there should be some normalization to allow for the predictable manipulation of the type used as a "Response" here. Some more tests are needed to replicate the other instances of Response fabrication.